src/psyclone/parse/module_manager.py

```diff
@@ -222,21 +243,145 @@ def ignores(self):
         '''
         return self._ignore_modules
 
-    # ------------------------------------------------------------------------
-    def get_module_info(self, module_name):
-        '''This function returns the ModuleInformation for the specified
+    def add_files(self, filepaths: Union[str, List[str], Set[str]]) -> None:
+        """Add a file to the list of files
```

Comment:
"Utility to add multiple files to the ModuleManager" perhaps?
Also, "Any file already found to be present in the ModuleManager is skipped."